sql/updates/world/2015_09_11_99_world_335.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE `trinity_string`
+SET `content_default` = 'Object GUID is: %s'
```

Comment:
if any other files are added by concatenation (cat command) after this file, the next file will start on the same line as =201; , so TC recommends that SQL files end with a NewLine or LF.